app/src/main/java/com/kickstarter/KSApplication.java

```diff
@@ -46,15 +46,17 @@ public void onCreate() {
     }
 
     JodaTimeAndroid.init(this);
-    FirebaseCrashlytics.getInstance().setCrashlyticsCollectionEnabled(true);
 
     this.component = DaggerApplicationComponent.builder()
       .applicationModule(new ApplicationModule(this))
       .build();
     component().inject(this);
 
-    FirebaseApp.initializeApp(this);
-    FirebaseAnalytics.getInstance(this).setAnalyticsCollectionEnabled(true);
+    if (FirebaseApp.getApps(getApplicationContext()).isEmpty() && !isInUnitTests()) {
```

Comment:
Question: Why do we need to check the app context?

Reply:
- FirebaseApp.getApps(getApplicationContext()).isEmpty() check is for initializing just once. 
- `getApplicationContext()` could be substituted by `this` i'm just accessing it with the contextWrapper method -> https://developer.android.com/reference/kotlin/android/content/ContextWrapper#getapplicationcontext

Reply:
Ah I see. Was this a change with the new version of Kotlin? Why was this changed?